nixos/release-combined.nix

```diff
@@ -91,6 +91,9 @@ in rec {
         (all nixos.tests.keymap.neo)
         (all nixos.tests.keymap.qwertz)
         (all nixos.tests.plasma5)
+        (all nixos.tests.kernel-copperhead)
```

Comment:
The copperhead kernel is in no way supported or release-critical so it shouldn't be part of the tested job.

Reply:
I will take it back out

Reply:
Fixed in 52341c03e80a594f49ddeaf86e78da6e834c6bb7